new_sem9/dz1.py

```diff
@@ -4,3 +4,65 @@
 # * Декоратор, запускающий функцию нахождения корней квадратного уравнения с каждой тройкой чисел из csv файла.
 # * Декоратор, сохраняющий переданные параметры и результаты работы функции в json файл.
 
+import csv
+import json
+import os
+from random import uniform, randint
+
+
+def find_quadratic_from_csv(func):
+    dic_res = {}
+    def wrapper():
+        with open('dz_new_sem9.csv', 'r', newline='') as f:
+            reader = csv.DictReader(f)
```

Comment:
Просто `reader` использовать было бы проще
```suggestion
            csv_reader = csv.reader(f)
```